amplify/backend/function/fetchMotionState/src/utils.js

```diff
@@ -33,9 +34,23 @@ const getVotingClient = async (colonyAddress) => {
   return colonyClient.getExtensionClient(Extension.VotingReputation);
 };
 
-const getLatestMotionState = async (colonyAddress, motionId) => {
-  const votingReputationClient = await getVotingClient(colonyAddress);
-  return votingReputationClient.getMotionState(motionId);
+const getLatestMotionState = async (
+  colonyAddress,
+  { nativeMotionId, createdBy },
+) => {
+  let motionState = MotionState.Null;
```

Comment:
Returning directly from the try/catch below would let you get rid of this variable (and you'd see straightaway what the catch returns without an extra comment)